test/images/agnhost/netexec/netexec.go

```diff
@@ -175,14 +175,15 @@ func main(cmd *cobra.Command, args []string) {
 		close(sigTermReceived)
 	}()
 
-	if delayShutdown > 0 {
-		go func() {
-			<-sigTermReceived
+	go func() {
+		<-sigTermReceived
+		if delayShutdown > 0 {
 			log.Printf("Sleeping %d seconds before terminating...", delayShutdown)
 			time.Sleep(time.Duration(delayShutdown) * time.Second)
-			os.Exit(0)
-		}()
-	}
+		}
+		// SIGTERM Exit Code 143
+		os.Exit(143)
```

Comment:
I have doubts about the exit codes, I've verified that previously, sending a SIGTERM the exit code is a 143
```
./agnhost netexec 
I0525 12:19:24.160983 1873100 log.go:184] Started HTTP server on port 8080
I0525 12:19:24.161146 1873100 log.go:184] Started UDP server on port  8081
Terminated
$ echo $?
143
```
if that is correct, the previous exit code to 0 was wrong

Reply:
for ref, this @rphillips comment here https://bugzilla.redhat.com/show_bug.cgi?id=1395663#c10